.github/CODEOWNERS

```diff
@@ -141,10 +134,8 @@ x-pack/examples/files_example @elastic/kibana-app-services
 /x-pack/test/apm_api_integration/ @elastic/apm-ui
 /src/plugins/apm_oss/  @elastic/apm-ui
```

Comment:
Not introduced by this PR but `apm_oss` plugin has been removed so we can remove this entry

Reply:
Cool, pushed up a change in [`989e029` (#138965)](https://github.com/elastic/kibana/pull/138965/commits/989e0295977c1277484cc17e9c8e3e7fbdc6c06b)

Reply:
Nice one thanks!